percy/util/ignoreRegion.js

```diff
@@ -0,0 +1,28 @@
+class IgnoreRegion {
+  constructor(top, bottom, left, right) {
+    if (top < 0 || bottom < 0 || left < 0 || right < 0) {
+      throw new Error('Only Positive integer is allowed!');
+    }
+
+    if (top >= bottom || left >= right) {
+      throw new Error('Invalid ignore region parameters!');
+    }
+
+    this.top = top;
+    this.bottom = bottom;
+    this.left = left;
+    this.right = right;
+  }
+
+  isValid(height, width) {
```

Comment:
```suggestion
  isValid(screenHeight, screenWidth) {
```
This avoids confusion between ignore element height vs screen height

Reply:
done